src/aria-snapshots/toast-test-aria-ts-variant-error-.yml

```diff
@@ -1,3 +1,3 @@
 - region "Notifications":
   - 'alert "Error: Label Description"':
-    - button "Close"
\ No newline at end of file
+    - button "Dismiss"
```

Comment:
Dismissing a toast is how we normally talk about getting rid of a notification. So I figured that's what screenreader users expect too.